src/Cassandra/Connections/ControlConnection.cs

```diff
@@ -680,8 +680,7 @@ private void HandleStatusChangeEvent(StatusChangeEventArgs e)
             //The address in the Cassandra event message needs to be translated
             var address = TranslateAddress(e.Address);
             ControlConnection._logger.Info("Received Node status change event: host {0} is {1}", address, e.What.ToString().ToUpper());
-            Host host;
-            if (!_metadata.Hosts.TryGet(address, out host))
+            if (!_metadata.Hosts.TryGet(address, out var host))
```

Comment:
This the only place you've used `out var`, is there a particular reason?

Reply:
Just simplification, I don't need to declare a local variable, since if it exists in my list, I will bring an instance of it.

Reply:
I'm sorry I don't understand. I'm asking specifically about `out var host` vs `out Host host` since everywhere else you used explicit type declarations and this is the only place where you used an implicit one.

Reply:
Sorry I got your question wrong, this was a point I would like to talk to you about, if we could start using `var` instead of implicit declaration.

This was specifically due to my impulse to use `var`.

Would you be okay with using var?

Reply:
Yes! I've been changing type declarations to `var` when I touch code that is not using `var` so +1 from me.